code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -175,7 +175,7 @@
 			new /obj/item/assembly/flash/hypnotic(src) // 7 TC
 			new /obj/item/storage/pill_bottle/lsd(src) // ~1 TC
 			new /obj/item/pen/sleepy(src) // 4 TC
-			new /obj/item/gun/ballistic/revolver/nagant(src) // 13 TC comparable to 357. revolvers
+			new /obj/item/gun/ballistic/revolver/sakhnomanni(src) // 13 TC comparable to 357. revolvers
```

Comment:
You know what, the code comment here made the claim it was a revolver equivalent but gosh was it absolutely not. You have a gun that can end up being incapable of being reloaded whatsoever once used up. At least with a basic revolver, you have the means to get more ammo.

It now actually is equivalent with this change (ignoring the fact that the only ammo you can get is weak to armor)